### ISSUE TITLE
Firefox: Rect width exceeds maximum margin: 3300

### ISSUE THREAD

**Issue by wijesijp** [cocos/cocos-engine#1959]
I am building an roulette game with Cocos Creator v1.5.1
I am using following code to load the roulette wheel sprites
Each sprite is 378x233
I switch the sprites manually to show wheel spinning

```
cc.loader.loadResDir(folderName, cc.SpriteFrame,
        function(completedCount ,totalCount ,item)
        {
            cc.log("progress " + completedCount*100/totalCount);
        },
        function (err, assets)
        {

            if (err) {
                cc.error(err.message || err);
                return;
            }

            for (var i = 0; i < assets.length; i++) {
                // load the frames in the animation to the array
                self._frameHolder[i] = assets[i];
            }             
    });
```

Sprite in the game is type : SIMPLE size mode : ROW

This works perfectly on all the browsers except Firefox.
On Firefox gives the error “Rect width exceeds maximum margin: 3300”

Instead of the wheel image it shows a white area.

This is the console error it is giving 

```
genLogFunc/< CCDebugger.js:367:12
_checkRect CCSpriteFrame.js:471:12
textureLoadedCallback CCSpriteFrame.js:295:20
_refreshTexture CCSpriteFrame.js:314:16
_loadTexture CCSpriteFrame.js:54:55
set CCSpriteFrame.js:76:13
loadCallback uuid-loader.js:132:20
loadDepends/< uuid-loader.js:151:24
[130]</proto.flowInDeps/queue< CCLoader.js:289:8
[139]</proto.allComplete loading-items.js:540:8
[139]</proto.itemComplete loading-items.js:727:8
[139]</LoadingItems.itemComplete loading-items.js:398:8
[141]</proto.flowOut pipeline.js:283:4
flow pipeline.js:86:16
flow/result< pipeline.js:63:20
[135]</Downloader.prototype.handle/syncRet< downloader.js:319:24
loadCallback downloader.js:95:12
```

I am loading 148 images for this animation. 
If reduce the number of images this error goes away.

Any idea why this is happening ?

**Comment by jareguo** [comment]
The texture is too large, try to resize to 2048 please.

**Comment by wijesijp** [comment]
Single image is 378 x 233

Yes size taken by all the images would be over 2048

But I am not building a single image.
I am just loading images from resource folder. 
I commented all other things, just loading the images is enough to give this error.

I tried to load images by grouping them into 20 from different folders 
I can load around 40 images without any error. 
If I load more the error is coming 

This is only happening on Firefox.

Any suggestions on what I should do?
